x-pack/plugins/cases/public/common/navigation/deep_links.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { AppDeepLink } from '../../../../../../src/core/public';
+import { DEFAULT_BASE_PATH, getCreateCasePath, getCasesConfigurePath } from './paths';
+
+export const casesDeepLinkIds = {
```

Comment:
Is there any reason why this cannot be an enum?

Reply:
Enums have some annoying limitations, such as this one that I couldn't solve: https://github.com/semd/kibana/blob/715437c64726efa2c2abe862d08106aee9217ceb/x-pack/plugins/security_solution/common/constants.ts#L86
Besides that, @XavierM once told me we should avoid using `enum` because it makes the bundle size bigger.

Reply:
As we discussed on slack, you are right. Better to have the object as const.